ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -297,8 +297,8 @@ Item {
                     chatSectionModule.prepareChatContentModuleForChatId(chatId)
                     let chatContentModule = chatSectionModule.getChatContentModule()
                     Global.openPopup(groupInfoPopupComponent, {
-                                         channelType: GroupInfoPopup.ChannelType.ActiveChannel,
-                                         channel: chatContentModule.chatDetails
+                                         chatContentModule: chatContentModule,
```

Comment:
why aren't we passing stores instead, this seems confusing as it seems like the module that was exposed already via nim, why not access them via the store itself?

Reply:
Create and issue and assigned it to you as discussed Boris -> https://github.com/status-im/status-desktop/issues/4823 :)